pykokkos/lib/ufuncs.py

```diff
@@ -133,13 +145,24 @@ def sqrt(view):
     """
     # TODO: support complex types when they
     # are available in pykokkos?
-    if len(view.shape) > 1:
-        raise NotImplementedError("only 1D views currently supported for sqrt() ufunc.")
-    if str(view.dtype) == "DataType.double":
-        pk.parallel_for(view.shape[0], sqrt_impl_1d_double, view=view)
-    elif str(view.dtype) == "DataType.float":
-        pk.parallel_for(view.shape[0], sqrt_impl_1d_float, view=view)
-    return view
+    if len(view.shape) > 2:
+        raise NotImplementedError("only up to 2D views currently supported for sqrt() ufunc.")
```

Comment:
I rebased and updated the error handling error to reflect that we now support up to 2D for this ufunc. Hopefully the CI agrees with my local testing here.